index.html

```diff
@@ -39,6 +39,7 @@ <h2 class="subtitle">Apache Kafka is a powerful, scalable, fault-tolerant distri
             <li><a href="kafka-console-consumer-producer-basics/kafka.html">Console producer and consumer basics</a></li>
             <li><a href="ccloud-produce-consume/kafka.html">Console producer and consumer basics with Confluent Cloud</a></li>
             <li><a href="kafka-console-consumer-primitive-keys-values/kafka.html">Console consumer with primitive keys and values</a></li>
+            <li><a href="console-consumer-produce-avro/kafka.html">Avro Console producer and consumer</a></li>
```

Comment:
IMO let's make these in line with the other ones here
```suggestion
            <li><a href="console-consumer-produce-avro/kafka.html">Console producer and consumer with Avro</a></li>
```